lib/old-roman-numerals.js

```diff
@@ -0,0 +1,32 @@
+const pry = require('pryjs');
+
+function oldRomanNumerals(number) {
+  let str = number.toString();
```

Comment:
we can assume here that the argument provided is indeed a number & no need to convert to string. I see where you were trying to go with this though. Maybe it would be better to store the roman numerial conversions in an object or array i.e. { 'M' : 1000, 'D': 500, ...} then looping through the values to see if the current number is larger, subtracting that number and continuing.